showpass/Showpass-E2E-tests
# Update element background-color validation

**Description:**
One of the css background-color validations began failing today.
Updated the element locator to more precisely pinpoint the button.

<img width="1713" alt="Screenshot 2023-10-10 at 3 34 42 PM" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/775e7160-043d-4156-8270-51a1ff8a9715">

**How to test:**
Run the following spec and ensure it passes cypress/e2e/user-settings/credits.cy.js

**Test execution results:**

All tests passed this time:

<img width="730" alt="Screenshot 2023-10-10 at 4 00 02 PM" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/7ffdd1fd-b8f8-465a-b964-4531a827fbbd">
